perllib/FixMyStreet/App/Controller/Admin/Users.pm

```diff
@@ -142,31 +142,60 @@ sub add : Local : Args(0) {
         $phone = $parsed_phone if $parsed_phone;
     }
 
-    my $existing_email = $email_v && $c->model('DB::User')->find( { email => $email } );
-    my $existing_phone = $phone_v && $c->model('DB::User')->find( { phone => $phone } );
-    if ($existing_email || $existing_phone) {
-        $c->stash->{field_errors}->{username} = _('User already exists');
-    }
-
-    my $user = $c->model('DB::User')->new( {
-        name => $c->get_param('name'),
-        email => $email ? $email : undef,
-        email_verified => $email && $email_v ? 1 : 0,
-        phone => $phone || undef,
-        phone_verified => $phone && $phone_v ? 1 : 0,
-        from_body => $c->get_param('body') || undef,
-        flagged => $c->get_param('flagged') || 0,
-        # Only superusers can create superusers
-        is_superuser => ( $c->user->is_superuser && $c->get_param('is_superuser') ) || 0,
-    } );
-    $c->stash->{user} = $user;
+    # Previously, an admin could add a staff user without a body (this is
+    # because the 'Staff' checkbox in the form could remain unchecked). A
+    # staff user without a body does not appear in the /admin/users list for
```

Comment:
Worse, they might not be able to access this user at all because it had no association with the cobrand (unless e.g. admin_user_domain had been set up).